project/src/pages/not-found-page/not-found-page.module.css

```diff
@@ -0,0 +1,18 @@
+.NotFound {
```

Comment:
А почему с большой?)
```suggestion
.notFound {
```